File: user-service/src/main/java/rs/edu/raf/repository/FavouriteRecipientRepository.java
package rs.edu.raf.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.FavouriteRecipient;

import java.util.List;

@Repository
public interface FavouriteRecipientRepository extends JpaRepository<FavouriteRecipient, Long> {
    List<FavouriteRecipient> findByClient_Id(Long clientId);
}

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/repository/FavouriteRecipientRepository.kt
package org.example.userservicekotlin.repository

import org.example.userservicekotlin.model.FavouriteRecipient
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FavouriteRecipientRepository : JpaRepository<FavouriteRecipient, Long> {
    fun findByClient_Id(clientId: Long): List<FavouriteRecipient>
}